forge-gui/res/cardsfolder/upcoming/rakdos_the_muscle.txt

```diff
@@ -0,0 +1,17 @@
+Name:Rakdos, the Muscle
+ManaCost:2 B B R
+Types:Legendary Creature Demon Mercenary
+PT:6/5
+K:Flying
+K:Trample
+T:Mode$ Sacrificed | ValidPlayer$ You | ValidCard$ Creature.Other | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you sacrifice another creature, exile cards equal to its mana value from the top of target player's library. Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
+SVar:TrigExile:DB$ Dig | DigNum$ X | ChangeNum$ All | ValidTgts$ Player | TgtPrompt$ Choose a player | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ UntilYourNextEndStep | SpellDescription$ Until your next end step, you may play those cards and mana of any type can be spent to cast those spells.
```

Comment:
SpellDescription useless

Reply:
Fair and done. 
Looking a bit more widely there do seem to be other scripts that need similar parameters excised: I got this one from Dead Man's Chest.